packages/manager/modules/pci/src/projects/project/data-processing/notebooks/data-processing-notebooks.controller.js

```diff
@@ -0,0 +1,70 @@
+import {
+  isNotebookRunning,
+  getClassFromStatus,
+} from '../data-processing.utils';
+import { DATA_PROCESSING_GUIDE_URL } from '../data-processing.constants';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    CucCloudMessage,
+    dataProcessingService,
+    ovhManagerRegionService,
+    atInternet,
+  ) {
+    this.cucCloudMessage = CucCloudMessage;
+    this.dataProcessingService = dataProcessingService;
+    this.ovhManagerRegionService = ovhManagerRegionService;
+    this.atInternet = atInternet;
+    this.isNotebookRunning = isNotebookRunning;
+    this.getClassFromStatus = getClassFromStatus;
+    this.guideUrl = DATA_PROCESSING_GUIDE_URL;
+  }
+
+  $onInit() {
+    this.subscribeToMessages();
+  }
+
+  refreshMessage() {
+    this.messages = this.messageHandler.getMessages();
+  }
+
+  subscribeToMessages() {
+    const channel = 'pci.projects.project.data-processing.notebooks';
+    this.cucCloudMessage.unSubscribe(channel);
+    this.messageHandler = this.cucCloudMessage.subscribe(channel, {
+      onMessage: () => this.refreshMessage(),
+    });
+  }
+
+  /**
+   * Start a notebook
+   * @param notebook Object the notebook we want to start
+   * @return {*|Promise<any>}
+   */
+  startNotebook(notebook) {
+    return this.dataProcessingService.startNotebook(
+      this.projectId,
+      notebook.id,
+    );
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  formatDuration(dt) {
+    // this method needs to use current instance of moment, so it cannot static
```

Comment:
**nitpick:** Make this function a static and avoid disabling the lint rules.
```suggestion
  static formatDuration(dt) {
    // this method needs to use current instance of moment, so it cannot static
```

Then in `$onInit` you can attach it to instance

`this.getFormatDuration = [Classname].formatDuration;`

P.S: Disabling lint rules is not an option.

Reply:
Sorry about that, I used old data-processing pages as a base and hasn't seen this one. I removed the lint rules disabler and set the method as static.